pipeline/outputs/prometheus-remote-write.md

```diff
@@ -13,6 +13,9 @@ Important Note: The prometheus exporter only works with metric plugins, such as
 | host                 | IP address or hostname of the target HTTP Server                                                                                                                                                                                                                       | 127.0.0.1 |
 | http_user            | Basic Auth Username                                                                                                                                                                                                                                                    |           |
 | http_passwd          | Basic Auth Password. Requires HTTP_user to be set                                                                                                                                                                                                                      |           |
+| AWS\_Auth                  | Enable AWS SigV4 authentication                                                                                                                                                                                                                                                                                                    | false     |
```

Comment:
the `FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP` adds more options than just these, please document them

Reply:
added